octobot_tentacles_manager/models/tentacle_type.py

```diff
@@ -0,0 +1,40 @@
+#  Drakkar-Software OctoBot-Tentacles-Manager
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+from os.path import sep
+
+
+class TentacleType:
+
+    def __init__(self, type_path, module_name=None):
+        self.path = type_path
+        self.module_name = module_name
+
+    @staticmethod
+    def from_import_path(root, import_path):
+        module_name = import_path.split(".")[-1]
```

Comment:
I think a best way to handle the path actions you are doing here is to use [pathlib](https://docs.python.org/3/library/pathlib.html)

Reply:
You can have something like : 
``` 
path = pathlib.Path(f"{root}{import_path}")
module_name = path.parts[1]
...
```

Reply:
We can use pathlib.Path for the ```self.path``` attribute but for this method, we have to operate on import path (such as ```tentacles.Evaluator.TA.momentum_evaluator.momentum_evaluator```) in this cas we want to extract ```momentum_evaluator``` and ```Evaluator.TA.momentum_evaluator```. I'm not sure replacing the import path by a pathlib.Path would improve the code.

Reply:
![image](https://user-images.githubusercontent.com/9078616/77847349-c5877280-71bc-11ea-8000-d3d8b0cf7648.png)

Reply:
Oh ok my bad :+1: